Config/ConfigHandler.cpp

```diff
@@ -12,14 +12,18 @@
 #include "../InputEvents/JoyButton.h"
 #include "../InputEvents/JoyAxisDirection.h"
 
-#include <boost/property_tree/detail/ptree_implementation.hpp>
-#include <boost/property_tree/ini_parser.hpp>
 #include <SDL2/SDL_keyboard.h>
 #include <SDL2/SDL_scancode.h>
 #include <SDL2/SDL_mixer.h>
 #include <exception>
 #include <iostream>
 #include <string>
+#include <json/json.h>
```

Comment:
I guess going from INI to JSON is OK. I know handling C++ dependencies on Windows is a pain though...